Dockerfile

```diff
@@ -14,31 +14,60 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-FROM registry.access.redhat.com/ubi9/ubi-minimal:9.3@sha256:bc552efb4966aaa44b02532be3168ac1ff18e2af299d0fe89502a1d9fabafbc5 AS downloads
+## Build
+
+FROM docker.io/library/golang:1.21 AS build
+
+ARG BUILD_LIST="darwin_amd64 darwin_arm64 linux_amd64 linux_arm64 linux_ppc64le linux_s390x windows_amd64"
+ARG TARGETOS
+ARG TARGETARCH
+
+WORKDIR /build
+
+# Copy just the mod file for better layer caching when building locally
+COPY go.mod go.sum .
+RUN go mod download
+
+# Now copy everything including .git
+COPY . .
+
+RUN ["/usr/bin/bash", "-c", "/build/build.sh \"${BUILD_LIST}\" \"${TARGETOS}\" \"${TARGETARCH}\""]
```

Comment:
Hmm why not just:

```
RUN /build/build.sh "${BUILD_LIST}" "${TARGETOS}" "${TARGETARCH}"
```

Reply:
The `RUN` command defaults to `/bin/sh`. I think I can change the default shell though so that isn't so messy.

Reply:
Why do you need bash for this simple command?

`build/build.sh` itself should run with what is specified in that file which is `/usr/bin/bash`.

Reply:
and you're right. thank you for that. I'll remove the SHELL